src/map.rs

```diff
@@ -311,7 +312,7 @@ where
             let mut sc = self.size_ctl.load(Ordering::SeqCst);
             if sc < 0 {
                 // we lost the initialization race; just spin
-                std::thread::yield_now();
+                core::sync::atomic::spin_loop_hint();
```

Comment:
I think we want to keep this `yield_now` if `std` is available.

Reply:
Also, let's include the argument for why this is okay from #26

Reply:
Is there a way to execute code when a feature is not present? Something like `#[not(cfg(feature = "std"))]` ...

Reply:
Almost!
```rust
#[cfg(not(feature = "std"))]
```